snapshot/spec/spec_helper.rb

```diff
@@ -10,3 +10,25 @@ module SpecHelper
 require 'snapshot'
 
 WebMock.disable_net_connect!(allow: 'coveralls.io')
+
+# Loading the XCode Project is really slow (>1 second)
+# fake it out for tests
+def fake_out_xcode_project_loading
+  fake_result = <<-EOS
+Information about project "Example":
+    Targets:
+        Example
+        ExampleUITests
+
+    Build Configurations:
+        Debug
+        Release
+
+    If no build configuration is specified and -scheme is not passed then "Release" is used.
+
+    Schemes:
+        Example
+        ExampleUITests
+EOS
+  allow_any_instance_of(FastlaneCore::Project).to receive(:raw_info).and_return fake_result
```

Comment:
I think we agree that this is a bit of a smell, and `rubocop-rspec` [seems to agree](https://github.com/nevir/rubocop-rspec/blob/master/lib/rubocop/cop/rspec/any_instance.rb#L6)... That said, this is a reasonable way of making this performant, and it probably has fewer gotchas than my instance caching way in `gym` and `scan`.

I think I would still prefer a `Project` data mocking approach that eliminates the `xctool` output and parsing entirely, but I'm on board with going this way for now, if you like.